packages/launch-editor/guess.js

```diff
@@ -56,6 +56,8 @@ module.exports = function guessEditor (specifiedEditor) {
         }
       }
     } else if (process.platform === 'win32') {
+      // compatible for path with Chinese characters
+      childProcess.execSync('chcp 65001');
       const output = childProcess
         .execSync(
           'powershell -NoProfile -Command "Get-CimInstance -Query \\"select executablepath from win32_process where executablepath is not null\\" | % { $_.ExecutablePath }"',
```

Comment:
```suggestion
      const output = childProcess
        .execSync(
          'powershell -NoProfile -Command "' +
            '[Console]::OutputEncoding=[Text.Encoding]::UTF8;' +
            'Get-CimInstance -Query \\"select executablepath from win32_process where executablepath is not null\\" | % { $_.ExecutablePath }' +
            '"',
```
I think this is better. This way

- it does not call a separate command. Calling each command has an overhead.
- it does not call `chcp` (which probably effects for all command executed later on).